packages/core/content-manager/admin/src/history/components/VersionInputRenderer.tsx

```diff
@@ -328,7 +333,25 @@ const VersionInputRenderer = ({
           {...props}
           hint={hint}
           disabled={fieldIsDisabled}
-          renderInput={(props) => <VersionInputRenderer {...props} shouldIgnoreRBAC={true} />}
+          renderLayout={(layoutProps) => {
+            // Components can only have one panel, so only save the first layout item
+            const [remainingFieldsLayout] = getRemaingFieldsLayout({
+              layout: [layoutProps.layout],
```

Comment:
The use of layoutProps.layout here feels circular. Isn't this the layout you are about to pass to layoutProps?

Reply:
I don't think I see the circular aspect here. I'm only passing a layout to the util so that it can know which fields are _not_ in that layout (e.g. the remaining ones)

Reply:
Isn't `remainingFieldsLayout` what you pass to the `layout` prop on `ComponentLayout` and then therefore the argument you receive in `renderLayout`?

Reply:
Not sure I 100% understand but it is normal if there is some sort of recursion here, the layout renders fields, some fields are components, therefore they render their own layout, etc

Reply:
👌